4337/hardhat.config.ts

```diff
@@ -25,16 +25,33 @@ if (PK) {
   }
 }
 
-if (['mainnet', 'rinkeby', 'kovan', 'goerli', 'ropsten', 'mumbai', 'polygon'].includes(argv.network) && INFURA_KEY === undefined) {
+if (['mainnet', 'goerli', 'mumbai', 'polygon'].includes(argv.network) && INFURA_KEY === undefined) {
   throw new Error(`Could not find Infura key in env, unable to connect to network ${argv.network}`)
 }
 
 import './src/tasks/local_verify'
 import './src/tasks/deploy_contracts'
 import './src/tasks/show_codesize'
 
-const primarySolidityVersion = SOLIDITY_VERSION || '0.8.21'
-const soliditySettings = !!SOLIDITY_SETTINGS ? JSON.parse(SOLIDITY_SETTINGS) : { optimizer: { enabled: true, runs: 10_000_000 } }
+const solidityVersion = SOLIDITY_VERSION || '0.8.21'
```

Comment:
```suggestion
const solidityVersion = SOLIDITY_VERSION || '0.8.23'
```

Reply:
I think we can also pump the solc version :)

Reply:
I would keep that change separate for a couple reasons:
- It means that we may potentially have the Safe4337Module end up on a different address that what is currently in the deployments
- This PR is intended to not change behaviour at all

Reply:
**Edit**: I think you are right… we are already developing for a 0.2 version anyway, so bumping makes sense.

Reply:
> I would keep that change separate for a couple reasons:

> It means that we may potentially have the Safe4337Module end up on a different address that what is currently in the deployments

Whoever wants to reproduce the deployment should use the release tag anyway :)